js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js

```diff
@@ -197,7 +197,7 @@ define([
                                     'Covariate ID': stat.covariateId,
                                     'Covariate name': stat.covariateName,
                                     'Count': stat.sumValue,
-                                    'Prevalence or %': stat.pct,
+                                    '%': stat.pct,
```

Comment:
Ok, great, but I woudln't make a table name a symbol Let's call it 'Percent' or 'Pct'..Probably 'percent' since the other columns are non-abbreviated.

Reply:
what about `Comparator prevalence or %`?

Reply:
Target Percent
Comparator Percent